proto/rig/operator/api/v1/pipeline/service.proto

```diff
@@ -39,10 +39,8 @@ enum ObjectType {
 message ObjectStatus {
   ObjectRef object_ref = 1;
   ObjectType type = 2;
-  google.protobuf.Timestamp updated_at = 3;
```

Comment:
What do we do instead of this timestamp?

Reply:
Likewise, is the state inferred from the conditions or what is the substitute?

Reply:
Yes, exactly :)